src/Codeception/Module/Db.php

```diff
@@ -41,85 +41,72 @@
+ * Connection is done by database drivers, which are stored in the `Codeception\Lib\Driver` namespace.
+ * Check out the drivers if you run into problems loading dumps and cleaning databases.
  *
- * ## Config
- *
- * * dsn *required* - PDO DSN
- * * user *required* - username to access database
- * * password *required* - password
- * * dump - path to database dump
- * * populate: false - whether the the dump should be loaded before the test suite is started
- * * cleanup: false - whether the dump should be reloaded before each test
- * * reconnect: false - whether the module should reconnect to the database before each test
- * * waitlock: 0 - wait lock (in seconds) that the database session should use for DDL statements
- * * ssl_key - path to the SSL key (MySQL specific, @see https://php.net/manual/de/ref.pdo-mysql.php#pdo.constants.mysql-attr-key)
- * * ssl_cert - path to the SSL certificate (MySQL specific, @see https://php.net/manual/de/ref.pdo-mysql.php#pdo.constants.mysql-attr-ssl-cert)
- * * ssl_ca - path to the SSL certificate authority (MySQL specific, @see https://php.net/manual/de/ref.pdo-mysql.php#pdo.constants.mysql-attr-ssl-ca)
- * * ssl_verify_server_cert - disables certificate CN verification (MySQL specific, @see https://php.net/manual/de/ref.pdo-mysql.php)
- * * ssl_cipher - list of one or more permissible ciphers to use for SSL encryption (MySQL specific, @see https://php.net/manual/de/ref.pdo-mysql.php#pdo.constants.mysql-attr-cipher)
- * * databases - include more database configs and switch between them in tests.
- * * initial_queries - list of queries to be executed right after connection to the database has been initiated, i.e. creating the database if it does not exist or preparing the database collation
- * * skip_cleanup_if_failed - Do not perform the cleanup if the tests failed. If this is used, manual cleanup might be required when re-running
- * ## Example
- *
- *     modules:
- *        enabled:
- *           - Db:
- *              dsn: 'mysql:host=localhost;dbname=testdb'
- *              user: 'root'
- *              password: ''
- *              dump: 'tests/_data/dump.sql'
- *              populate: true
- *              cleanup: true
- *              reconnect: true
- *              waitlock: 10
- *              skip_cleanup_if_failed: true
- *              ssl_key: '/path/to/client-key.pem'
- *              ssl_cert: '/path/to/client-cert.pem'
- *              ssl_ca: '/path/to/ca-cert.pem'
- *              ssl_verify_server_cert: false
- *              ssl_cipher: 'AES256-SHA'
- *              initial_queries:
- *                  - 'CREATE DATABASE IF NOT EXISTS temp_db;'
- *                  - 'USE temp_db;'
- *                  - 'SET NAMES utf8;'
+ * ## Example `Functional.suite.yml`
+ * ```yaml
+ * modules:
+ *     enabled:
+ *         - Db:
+ *             dsn: 'mysql:host=localhost;dbname=testdb'
+ *             user: 'root'
+ *             password: ''
+ *             dump: 'tests/_data/dump.sql'
```

Comment:
```suggestion
 *             dump: 'tests/Support/Data/dump.sql'
```

IIRC the default path is `tests/Support/Data`